spec/integration/api/v1/transactions_controller_spec.rb

```diff
@@ -102,7 +101,7 @@
 
         All parameters are optional with the following constraints and defaults:
            * When leaving the `debtor` or `creditor` empty, this will default to the Zeus user.
-           * At least one of `euros` and `cents` needs to be filled in
+           * `cents` must be an integer, never a string or float
```

Comment:
This doesn't match the behaviour in the controller :thinking:

Reply:
(as in, it's allowed to be a string representing a number)

Reply:
Ah yes, fixed.